gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -1033,3 +1039,96 @@ message Shape {
   // The following extension IDs are reserved for private use by any organization.
   extensions 9000 to 9999;
 }
+
+// Describes a stop which is served by trips. All fields are as described in the GTFS-Static specification.
+message Stop {
+  enum WheelchairBoarding {
+    UNKNOWN = 0;
+    AVAILABLE = 1;
+    NOT_AVAILABLE = 2;
+   }
+
+  required string stop_id = 1;
```

Comment:
Apologies if this was already mentioned, I wanted to call out that official guidance around protocol buffers is to *not* introduce new fields that use `required`, per https://protobuf.dev/programming-guides/proto2/#field-labels

There are compelling reasons why, some of which are discussed in the link, so even if this is technically a "safer" addition at the moment, I would keep it optional here regardless.  I know GTFS-RT uses required fields elsewhere, but I think that was from a time when the guidance was less clear.

Reply:
(See also the existing comment on Shape.shape_id)

Reply:
Yes thanks for pointing that out. Fixing